src/clarity.rs

```diff
@@ -489,6 +492,18 @@ fn consume_arg(
 }
 
 pub fn invoke_command(invoked_by: &str, args: &[String]) {
+    let mut args = Vec::from(args);
+
+    let coverage_folder =
+        if let Some(coverage_flag_index) = args[1..].iter().position(|arg| arg == "-c") {
```

Comment:
Do you want to use `consume_arg()` here instead? It does all the same kinds of lifting.